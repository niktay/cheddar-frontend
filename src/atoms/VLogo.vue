<template>
  <img class="a-VLogo" src="@/assets/logo.svg" alt="Cheddar Logo">
</template>

<script>
export default {};
</script>

<style class="scss">
.a-VLogo {
  max-height: 2.2rem;
}
</style>