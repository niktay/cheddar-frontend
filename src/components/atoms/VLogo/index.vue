<template>
  <img class="a-VLogo" src="../../../assets/logo.svg" alt="Cheddar Logo">
</template>

<script>
import CheddarLogo from "@/assets/logo.svg";

export default {
  data: function() {
    return {
      logo: CheddarLogo
    };
  }
};
</script>

<style class="scss">
.a-VLogo {
  max-height: 2.2rem;
}
</style>
