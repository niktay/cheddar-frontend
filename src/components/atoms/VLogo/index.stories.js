import { storiesOf } from "@storybook/vue";
import VLogo from ".";

storiesOf("Atom - VLogo", module).add("default", () => ({
  components: { VLogo },

  template: `<VLogo />`
}));
