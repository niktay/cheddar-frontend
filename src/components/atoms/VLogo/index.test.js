import { mount } from "@vue/test-utils";
import VLogo from ".";

describe("VLogo component", () => {
  it("should render correctly", () => {
    const comp = mount(VLogo);

    expect(comp.html()).toMatchSnapshot();
  });
});
