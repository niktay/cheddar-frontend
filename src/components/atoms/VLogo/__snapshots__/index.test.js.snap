// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VLogo component should render correctly 1`] = `<img src="../../../assets/logo.svg" alt="Cheddar Logo" class="a-VLogo">`;
